pkg/osvscanner/vulnerability_result.go

```diff
@@ -59,11 +59,14 @@ func buildVulnerabilityResults(
 		pkg.DepGroups = rawPkg.DepGroups
 
 		if len(vulnsResp.Results[i].Vulns) > 0 {
-			includePackage = true
-			pkg.Vulnerabilities = vulnsResp.Results[i].Vulns
-			pkg.Groups = grouper.Group(grouper.ConvertVulnerabilityToIDAliases(pkg.Vulnerabilities))
-			for i, group := range pkg.Groups {
-				pkg.Groups[i].MaxSeverity = output.MaxSeverity(group, pkg)
+			configToUse := configManager.Get(r, rawPkg.Source.Path)
```

Comment:
note: it might make sense to pull this out of the condition since we do the exact same thing for license scanning below

but I've purposely not done that for now as I think the performance impact will be pretty minimal, it should be easily doable in a follow-up PR, and there might be a reason why we don't want to do that...